frontend/src/metabase/components/SingleSelectListField/SingleSelectListField.unit.spec.js

```diff
@@ -20,7 +20,7 @@ function renderValue(fields, formatOptions, value, options) {
     <ValueComponent
       value={value}
       column={fields[0]}
-      maximumFractionDigits={20}
+      maximum_fraction_digits={20}
```

Comment:
We seem to still use the snake case here?

Reply:
Ah, good catch. I think I searched for uses of `_significant_digits` but not `_fraction_digits`.
Fixed.